bigbluebutton-html5/imports/ui/components/button/component.jsx

```diff
@@ -117,21 +117,25 @@ export default class Button extends BaseButton {
       circle,
       hideLabel,
       label,
-      'aria-label' : ariaLabel
+      'aria-label': ariaLabel,
     } = this.props;
 
     const renderFuncName = circle ? 'renderCircle' : 'renderDefault';
 
     if (hideLabel) {
-      const tooltipLabel = label ? label : ariaLabel;
+      const tooltipLabel = label || ariaLabel;
 
-      return (
+      const isMobile
+      = /Android|iPhone|iPad|Windows Phone|Opera Mini|IEMobile|Mobile/i.test(navigator.userAgent);
```

Comment:
We want to disable it for any TOUCH device, not mobile. 

`const supportsTouch = 'ontouchstart' in window || navigator.msMaxTouchPoints;`